root/etc/e-smith/templates/etc/rspamd/local.d/worker-normal.inc/10base

```diff
@@ -0,0 +1,6 @@
+# socket for worker-normal
+bind_socket = "/var/run/rspamd/worker-normal mode=0770 owner=_rspamd group=mail";
```

Comment:
The documentation says to disable worker-normal in self-scan mode: does it apply here?

https://rspamd.com/doc/workers/rspamd_proxy.html#self-scan-mode